java/adapter/jdbc/src/main/java/org/apache/arrow/adapter/jdbc/consumer/JdbcConsumer.java

```diff
@@ -44,4 +44,10 @@
    */
   void resetValueVector(T vector);
 
+  /**
```

Comment:
please comment here, that we expose this method to optimize for the non-null case and that in tests, it by override to false, it improves microbenchmarks by x%

Reply:
Good point. I have added the comment accordingly. Thanks.